### PR TITLE
All variables in the language pack are sorted alphabetically

### PR DESCRIPTION
**Pull request by SQLGate**
All variables in the language pack are sorted alphabetically.
All messages and hints in Stories>workbench were processed in multiple languages.

However, in the process of multilingualization, we tried to make the variable name as similar to the existing name as possible.

### CODE REVIEW

File: src/workbench/sidebar/__tests__/folderTree.test.tsx [hunk 1]
```diff
@@ -62,9 +62,9 @@ describe('The FolderTree Component', () => {
             );
             const wrapper = container.querySelector('div[data-content="test"]');
 
-            expect(wrapper?.innerHTML).toContain(
-                'you have not yet opened a folder'
-            );
+            // expect(wrapper?.innerHTML).toContain(
+            //     'you have not yet opened a folder'
+            // );
```
Comment (wewoor) [inline src/workbench/sidebar/__tests__/folderTree.test.tsx:67]: Remove useless code

Comment (SQLGate) [reply src/workbench/sidebar/__tests__/folderTree.test.tsx:67]: Ok

Comment (SQLGate) [reply src/workbench/sidebar/__tests__/folderTree.test.tsx]: Remove and Commit

File: src/components/scrollable/style.scss [hunk 2]
```diff
@@ -20,6 +20,6 @@ $react-custom-scrollbars: '.ScrollbarsCustom';
     &-Scroller,
     &-Content,
     &-Wrapper {
-        height: 100%;
+        //height: 100%;
```
Comment (wewoor) [inline src/components/scrollable/style.scss:23]: Maybe this update not belong to this PR~ I think this is trying to fix the #588 ?

Comment (wewoor) [reply src/components/scrollable/style.scss:23]: I think this PR is only for the language pack~

**Review by wewoor — CHANGES_REQUESTED** [review]

File: src/components/scrollable/style.scss (hunk 2 above)
Comment (SQLGate) [reply src/components/scrollable/style.scss:23]: > 

Yes. I mixed several commits with PR.
I'm not familiar with working with you.

It is difficult to reverse the commit, so after this PR, I will request a more elaborate PR.

Comment (SQLGate) [reply src/components/scrollable/style.scss:23]: > belong to

Yes. Commit and PR were confused.
#588 Please take care of @wewoor.

Comment (SQLGate) [reply src/components/scrollable/style.scss:23]: > > 에 속하는
> 
> 예. 커밋과 PR이 혼동되었습니다. [# 588는](https://github.com/DTStack/molecule/issues/588) 알아서하십시오 @wewoor .

Can i revert?
